usr/lib/ica_s390_stdll/ica_specific.c

```diff
@@ -2588,11 +2976,42 @@ static CK_RV ica_specific_rsa_decrypt(STDLL_TokData_t *tokdata,
     crtKey = ex_data->crtKey;
     modexpoKey = ex_data->modexpoKey;
 
+    bn_ctx = BN_CTX_new();
+    if (bn_ctx == NULL) {
+        TRACE_ERROR("BN_CTX_new failed\n");
+        rc = CKR_FUNCTION_FAILED;
+        goto done;
+    }
+
+    if (ex_data->blinding == NULL || ex_data->blinding_mont_ctx == NULL) {
+        rc = ica_blinding_setup(tokdata, key_obj, ex_data, bn_ctx);
+        if (rc != CKR_OK) {
+            TRACE_ERROR("ica_setup_blinding failed\n");
```

Comment:
Typo ica_setup_blinding -> ica_blinding_setup

Reply:
you have good eyes ;-) 
will fix